api/v1alpha1/iochaos_types.go

```diff
@@ -84,7 +84,7 @@ type IoChaosSpec struct {
 	// such as "300ms", "-1.5h" or "2h45m".
 	// Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
 	// +optional
-	Duration string `json:"duration"`
+	Duration *string `json:"duration,omitempty"`
```

Comment:
why using pointer here? you can use empty to check the duration directly

Reply:
Using pointer here is to make chaos API more user-friendly. If users need the chaos action always perform after they apply the chaos API, it is more reasonable that they omit the `scheduler` and the `duration` instead of leaving those as empty.